
Sat, 27 Nov 2010 01:48:51 GMT

COMMAND LINE:
/usr/local/src/pypy-1.4/bin/pypy nbody.pypy-4.pypy 500000

PROGRAM FAILED 


PROGRAM OUTPUT:

Traceback (most recent call last):
  File "app_main.py", line 33, in run_toplevel
  File "nbody.pypy-4.pypy", line 51, in <module>
    PAIRS = list(itertools.combinations(SYSTEM, 2))
AttributeError: 'module' object has no attribute 'combinations'
